internal/testplanet/satellite.go

```diff
@@ -188,6 +189,12 @@ func (system *SatelliteSystem) Run(ctx context.Context) (err error) {
 	group.Go(func() error {
 		return errs2.IgnoreCanceled(system.API.Run(ctx))
 	})
+	group.Go(func() error {
+		return errs2.IgnoreCanceled(system.Repairer.Run(ctx))
+	})
+
+	system.Peer.Repair.Repairer.Loop.Stop()
```

Comment:
could we add a comment why we are stopping this?

Reply:
Also there should be no need to stop this, since the repairer should be able to run concurrently.